app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 32
+    number_of_parts = 6
+    split = split_integer(value, number_of_parts)
+    assert sum(split) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 16
+    number_of_parts = 4
+    every_integer = value // number_of_parts
+    split = split_integer(value, number_of_parts)
+    for number in split:
+        assert number == every_integer
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 16
+    number_of_parts = 1
+    split = split_integer(value, number_of_parts)
+    assert split == [value]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    split = split_integer(17, 4)
+    assert split == sorted(split)
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    value = 4
+    number_of_parts = 6
+    difference = number_of_parts - value
+    split = split_integer(value, number_of_parts)
+    assert split == [0] * difference + [1] * value
```

Comment:
Look at how asserts work. You should state what the result of the test will be, not write the test itself.